crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -158,8 +159,17 @@ impl WrapEvtSender {
             true,
             {
                 self.0.call_remote(
-                    dna_hash, from, signature, to_agent, zome_name, fn_name, cap_secret, payload,
-                    nonce, expires_at,
+                    dna_hash,
+                    from,
+                    zome_call_payload,
+                    signature,
```

Comment:
I would have loved to use `ZomeCallParamsSigned` here, but I couldn't find a configuration where the type is accessible from here as well as `holochain_conductor_api`.

Reply:
These libraries don't have `holochain_types` as a common dependency?

Reply:
Oh, but I guess the type needs access to things that shouldn't be in `holochain_types` as part of its implementation